svn/common.py

```diff
@@ -49,10 +51,13 @@ def run_command(self, subcommand, args, success_code=0,
 
         _logger.debug("RUN: %s" % (cmd,))
 
+        environment_variables = copy.deepcopy(os.environ)
```

Comment:
A deepcopy is not necessary. This is also a simple dictionary. Just use `{}.copy()`.